Bindings/Java/tests/TestTables.java

```diff
@@ -58,6 +58,14 @@ public static void test_DataTable() {
         System.out.println(table);
         // Clone table.
         DataTable tableClone = table.clone();
+        // Reorder columns of the table.
```

Comment:
It isn't clear/obvious how you're confirming that the column reordering is working properly. For example, if the intention is to scramble in L63–65 and unscramble in L66–68, then the test will still pass if `table.reorderColumns()` does nothing (e.g., if somehow, sometime in the future, the method ends up throwing a nonfatal exception that isn't caught).